app/views/upload.scala.html

```diff
@@ -22,7 +22,7 @@ <h1 class="govuk-heading-xl">Upload</h1>
                         Upload a folder
                     </label>
                     <input type="hidden" class="file-id-data" name="file-id-data" />
-                    <input type="file" id="file-upload" name="files" class="govuk-file-upload" webkitdirectory>
+                    <input required type="file" id="file-upload" name="files" class="govuk-file-upload" webkitdirectory>
```

Comment:
It looks like this change gets reverted in the next commit, so it would be nice to just delete these commits altogether to save confusion.

Reply:
REsolved